src/qt/bitcoin.cpp

```diff
@@ -681,13 +680,21 @@ int main(int argc, char *argv[])
     //   IMPORTANT if it is no longer a typedef use the normal variant above
     qRegisterMetaType< CAmount >("CAmount");
 
+    /// 2. Parse command-line options. Command-line options take precedence:
+    try {
+        ParseParameters(argc, argv, AllowedArgs::BitcoinQt);
+    } catch (const std::exception& e) {
+        QMessageBox::critical(0, QObject::tr("Bitcoin Classic"),
```

Comment:
Shouldn't this be "Bitcoin Unlimited" for us? Wasn't there central definition for the client somewhere anyways?